servers/jax-rs-testextension/src/main/java/org/projectnessie/jaxrs/ext/NessieJaxRsExtension.java

```diff
@@ -125,25 +127,36 @@ public void beforeEach(ExtensionContext extensionContext) {
   public boolean supportsParameter(
       ParameterContext parameterContext, ExtensionContext extensionContext)
       throws ParameterResolutionException {
-    return parameterContext.isAnnotated(NessieUri.class)
+    return super.supportsParameter(parameterContext, extensionContext)
+        || parameterContext.isAnnotated(NessieUri.class)
         || parameterContext.isAnnotated(NessieSecurityContext.class)
         || parameterContext.isAnnotated(NessieAccessChecker.class);
   }
 
-  @Override
-  public Object resolveParameter(
-      ParameterContext parameterContext, ExtensionContext extensionContext)
-      throws ParameterResolutionException {
+  private EnvHolder getEnv(ExtensionContext extensionContext) {
```

Comment:
i think this helper can be used in 3 more places

Reply:
ah, missed that - updated